boards/nxp/lpcxpresso55s69/lpcxpresso55s69_lpc55s69_cpu0_ns_defconfig

```diff
@@ -1,20 +1,16 @@
 #
-# Copyright (c) 2019, NXP
+# Copyright 2019,2024 NXP
 #
 # SPDX-License-Identifier: Apache-2.0
 #
 
-CONFIG_SOC_SERIES_LPC55XXX=y
-CONFIG_SOC_LPC55S69_CPU0=y
-CONFIG_BOARD_LPCXPRESSO55S69_CPU0=y
 CONFIG_CONSOLE=y
 CONFIG_UART_CONSOLE=y
 CONFIG_SERIAL=y
 CONFIG_UART_INTERRUPT_DRIVEN=y
 CONFIG_GPIO=y
 CONFIG_PINCTRL=y
 
-# TFM sets up MPU_NS, can't correctly change this configuration yet
 CONFIG_ARM_MPU=n
```

Comment:
Not requesting changes here but you can have a common file i.e. `lpcxpresso55s69_defconfig` which defines common things for all the CPUs of this board, and just put the additional changes needed for each core in the extra _defconfig files, or remove any that don't need any changes